### DISCUSSION TITLE
iframe Integration Removed

### DISCUSSION THREAD

**Discussion by wallzero** [music-assistant/support#2344] [Q&A]
[The docker instructions](https://music-assistant.io/faq/how-to/#get-the-ma-icon-in-the-ha-sidebar) refer to using an iframe integration that is [no longer supported](https://www.home-assistant.io/more-info/removed-integration).

**Comment by OzGav** [comment]
Thanks. This is in the next version of the docs to be pushed in the next 24 hours.
